src/containers/Flow/FlowList/FlowList.tsx

```diff
@@ -157,19 +164,22 @@ export const FlowList = () => {
     keywords,
     lastChangedAt,
     lastPublishedAt,
-    isPinned,
+    tag,
     roles,
+    isPinned,
   }: any) => ({
     pin: displayPinned(isPinned),
     name: getName(name, keywords, roles),
     lastPublishedAt: getLastPublished(lastPublishedAt, t('Not published yet')),
+    label: getLabel(tag),
     lastChangedAt: getDate(lastChangedAt, t('Nothing in draft')),
   });
 
   const columnNames = [
-    { name: 'is_pinned', label: '', sort: true, order: 'desc' },
+    { label: '' },
```

Comment:
Why did we remove this?

Reply:
The icon was not present in the design. Resolved